homeassistant/components/yeelight/services.yaml

```
@@ -1,96 +1,190 @@
     brightness:
-      description: The brightness value to set (1-100).
-      example: 50
+      name: Brightness
+      description: The brightness value to set.
+      selector:
+        number:
+          min: 0
+          max: 100
 set_color_temp_scene:
+  name: Set color temperature scene
   description: Changes the light to the specified color temperature. If the light is off, it will be turned on.
+  target:
+    entity:
+      integration: yeelight
+      domain: light
   fields:
-    entity_id:
-      description: Name of the light entity.
-      example: "light.yeelight"
     kelvin:
+      name: Kelvin
       description: Color temperature for the light in Kelvin.
       example: 4000
+      selector:
+        number:
+          min: 0
+          max: 1000000
     brightness:
-      description: The brightness value to set (1-100).
-      example: 50
+      name: Brightness
+      description: The brightness value to set.
+      selector:
+        number:
+          min: 0
+          max: 100
 set_color_flow_scene:
+  name: Set color flow scene
   description: starts a color flow. If the light is off, it will be turned on.
+  target:
+    entity:
+      integration: yeelight
+      domain: light
   fields:
-    entity_id:
-      description: Name of the light entity.
-      example: "light.yeelight"
     count:
+      name: Count
       description: The number of times to run this flow (0 to run forever).
-      example: 0
+      default: 0
+      selector:
+        number:
+          min: 0
+          max: 100
     action:
-      description: The action to take after the flow stops. Can be 'recover', 'stay', 'off'. (default 'recover')
+      name: Action
+      description: The action to take after the flow stops.
       example: "stay"
+      default: 'recover'
+      selector:
+        select:
+          options:
+            - 'off'
+            - 'recover'
+            - 'stay'
     transitions:
+      name: Transitions
       description: Array of transitions, for desired effect. Examples https://yeelight.readthedocs.io/en/stable/flow.html
       example: '[{ "TemperatureTransition": [1900, 1000, 80] }, { "TemperatureTransition": [1900, 1000, 10] }]'
+      selector:
+        object:
 set_auto_delay_off_scene:
+  name: Set auto delay off scene
   description: Turns the light on to the specified brightness and sets a timer to turn it back off after the given number of minutes. If the light is off, Set a color scene, if light is off, it will be turned on.
+  target:
+    entity:
+      integration: yeelight
+      domain: light
   fields:
-    entity_id:
-      description: Name of the light entity.
-      example: "light.yeelight"
     minutes:
-      description: The minutes to wait before automatically turning the light off.
+      name: Minutes
+      description: The time to wait before automatically turning the light off.
       example: 5
+      selector:
+        number:
+          min: 0
+          max: 1440
```

The value is limited to this range
```suggestion
          min: 1
          max: 60
```

---

Is the `max` necessary here? 
```suggestion
          min: 1
          unit_of_measurement: minutes
          mode: box
```

---

@TheZoker, yes, max is required for the selector schema. Additionally, this specific service limits it to 60.